workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSReceiveSystem.cs

```diff
@@ -157,120 +112,17 @@ internal void RemoveEntity(EntityId entityId)
             }
 
             Profiler.BeginSample("OnRemoveEntity");
-            WorldCommands.DeallocateWorldCommandRequesters(EntityManager, entity);
             EntityManager.DestroyEntity(worker.EntityIdToEntity[entityId]);
             worker.EntityIdToEntity.Remove(entityId);
             Profiler.EndSample();
         }
 
         internal void OnDisconnect(string reason)
```

Comment:
Should this one be made `private` as well?

Reply:
Yes